evap/staff/tests/test_views.py

```diff
@@ -1920,6 +1920,38 @@ def test_evaluation_create(self):
         form.submit()
         self.assertEqual(Evaluation.objects.get().name_de, "lfo9e7bmxp1xi")
 
+    def _set_valid_form(self, form):
```

Comment:
factoring out the helper is nice -- but we should also use it in `test_evaluation_create` then :D